src/events/message/commands/fun/rankup.ts

```diff
@@ -160,6 +162,9 @@ export default async function (message: Message, args: CommandArg[], opts: Optio
     [username] = result.data;
   }
 
+  // If current channel is not #bot-playground, redirect them to that channel
+  if ((message.channel as TextChannel).name !== "bot-playground") return send(REDIRECT);
```

Comment:
This should be at the beginning of the function. It shouldn't show the usage.

Reply:
Should be fixed now, please confirm